src/Listbox.tsx

```diff
@@ -101,21 +97,34 @@ export class Listbox extends Component<Props, State> {
     })
   }
 
+  getFocusValue = (): string | null => {
+    const activeValue = this.props.value
+    // Set active value to be the first option
+    // in the list if no item is selected.
+    // https://www.w3.org/TR/wai-aria-practices/#listbox_kbd_interaction
+    if (!activeValue){ return this.state.values[0] }
+    return Array.isArray(activeValue) ? activeValue[0] : activeValue
+  }
+
   close = (): void => {
     this.setState({ isOpen: false }, () => { this.state.listboxButtonRef?.focus() })
   }
 
   select = (value: string): void => {
-    this.props.onChange(value)
-    process.nextTick(() => {
-      this.close()
-    })
+    if (!this.props.multiselect) {
+      this.props.onChange(value)
+      process.nextTick(() => {
+        this.close()
+      })
+    } else if (this.props.value !== null && Array.isArray(this.props.value)) {
+      this.props.onChange([value, ...this.props.value])
+    }
   }
 
   focus = (value: string | null): void => {
     this.setState({ activeItem: value }, () => {
       if (value === null){ return }
-      this.state.listboxListRef?.children[this.state.values.indexOf(this.state.activeItem)].scrollIntoView({ block: "nearest" })
+      this.state.listboxListRef?.children[this.state.values.indexOf(this.getFocusValue())].scrollIntoView({ block: "nearest" })
```

Comment:
🤦  This should use value instead of the `getFocusValue` function.

I'll rename `getFocusValue` to `getDefaultFocusValue` to make it clearer what that function does.

Reply:
Done now